opencti-platform/opencti-graphql/tests/01-unit/domain/csv-bundler-test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, it, expect } from "vitest";
+import { bundleProcess } from "../../../src/parser/csv-bundler";
+import { ADMIN_USER, testContext } from "../../utils/testQuery";
+import type {StixBundle, StixDomainObject } from "../../../src/types/stix-common";
+import {
+  indicatorsWithExternalReferencesCsvContent,
+  indicatorsWithExternalReferencesCsvMapper
+} from "../../data/csv-bundler/external-references-constants";
+import type {CsvMapperParsed} from "../../../src/modules/internal/csvMapper/csvMapper-types";
+
+describe('CSV bundler', () => {
+  describe('Embedded properties', () => {
+    it('Should list external references', async() => {
+      const indicatorsWithExternalReferencesExpectedBundle: StixBundle = {
```

Comment:
I'm wondering why do we need this whole bundle if we only use it to compare the external references with the result of the csv mapper ? 
It took me a bit of time to understand how the test works, the bundle being an expected result, and the inputs we use (csvmapper & csv input) are defined in external-references-constants.ts

Reply:
This assertion is based on the bug we had before.
This way, we will know the root cause if it occurs again.
Since it was fixed by another evolution, I wanted to document it.

Reply:
I'm not sure it's the same remark as @SouadHadjiat but in the `expect` clause, you only check the `external_references`. So why this complete bundle object ?

Now, it might make sense to expect a deep equal on the full bundle, if this was not yet covered in the tests before.

Reply:
Sure, let me do that